src/components/Index.js

```diff
@@ -0,0 +1,51 @@
+import { createSuitMixin } from '../mixins/suit';
+import { createWidgetMixin } from '../mixins/widget';
+// TODO: should InstantSearch expose a fake connector instead? Would be more consistent.
+import indexWidget from 'instantsearch.js/es/widgets/index/index';
+
+// wrapped in a dummy function, since indexWidget doesn't render
+const connectIndex = () => indexWidget;
```

Comment:
This is fine, but maybe it should live in InstantSearch instead for consistency?

Reply:
Yes – this will become more obvious once we decouple InstantSearch.js from the connectors.